lib/tasks/copy_feedback_tables_into_one_consolidated_table.rake

```diff
@@ -0,0 +1,75 @@
+namespace :consolidate_feedback_tables do
+  desc "Copy data from feedback tables into one table"
+  task consolidate_feedback_tables: :environment do
+    AccountFeedback.all.in_batches(of: 100).each_record do |account_feedback|
```

Comment:
Instead of `.all.in_batches(of: 100).each_record` you can just use `find_each`
https://api.rubyonrails.org/v6.1.0/classes/ActiveRecord/Batches.html#method-i-find_each